app/templates/request-quote-view.jade

```diff
@@ -0,0 +1,147 @@
+extends /templates/base
+
+block content
+  form.form
+    h1.text-center(data-i18n="teachers_quote.title")
+    p.text-center(data-i18n="teachers_quote.subtitle")
+
+    .row
+      .col-sm-offset-2.col-sm-4
+        .form-group
+          label.control-label(data-i18n="general.name") 
+          input.form-control(name="name")
+          
+      .col-sm-4
+        .form-group
+          label.control-label(data-i18n="general.email")
+          input.form-control(name="email")
+    
+    .row
+      .col-sm-offset-2.col-sm-4
+        .form-group
+          label.control-label
+            span(data-i18n="teachers_quote.phone_number")
+            span.spl.text-muted(data-i18n="signup.optional")
+          .help-block
+            em.text-info(data-i18n="teachers_quote.phone_number_help")
+          input.form-control(name="phoneNumber")
+          
+      .col-sm-4
+        .form-group
+          label.control-label(data-i18n="teachers_quote.role_label")
+          .help-block
+            em.text-info(data-i18n="teachers_quote.role_help")
+          select.form-control(name="role")
```

Comment:
We should default to unselected, instead of Teacher.  We should get better data if you have to select an option.